Automattic/jetpack
# Pay with Paypal block: Increased the size of the Pay with Paypal button

Increased the size of the Pay with Paypal button in Pay with Paypal block in order to display the card icons shown below the button.

Besides this, I've also changed the icons that are used in Gutenberg so that it better reflect the actual size rendered on the front-end.




Fixes #17324

#### Changes proposed in this Pull Request:

* Mark the button as responsive
* Changed the png icons used in Gutenberg

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Atomic/Jetpack: switch to this branch using Jetpack Beta
* WPCOM:  Run `arc patch-image D73251` to apply the patch, since it contains images. 
* Go to Posts > Add new and add the Pay with Paypal button
* Save the post.
* Make sure that the button is larger in Gutenberg
* Preview the newly created post and make sure that the card icons beneath the button are not truncated.

**Before**:
![Screenshot 2022-01-18 at 17 00 12](https://user-images.githubusercontent.com/995955/149962179-9978b9f9-6a7a-4bbe-ad67-6bba03f8bb73.png)


**After**:
![Screenshot 2022-01-18 at 17 00 35](https://user-images.githubusercontent.com/995955/149962204-a1c9bf5a-11a0-4de3-9990-883c88728c1c.png)
